bundle-size-chart/src/app.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createObjectCsvStringifier} from 'csv-writer';
+import dotenv from 'dotenv';
+import express from 'express';
+import process from 'process';
+import Octokit from '@octokit/rest';
+import {Storage} from '@google-cloud/storage';
+
+const CRON_TIMEOUT_MS = 600000; // 10 minutes, the GAE cron timeout limit.
+const MAX_COMMIT_DAYS = 365;
+const GCLOUD_STORAGE_BUCKET = 'amp-bundle-size-chart';
+
+dotenv.config();
+
+const app = express();
+app.use(express.static('static'));
+
+app.get('/_cron', async (request, response) => {
```

Comment:
Would like to see methods split out here with corresponding unit tests. Not needed right away.

Reply:
I very much doubt I'll ever write tests for this 😅